yarn.lock

```diff
@@ -5,7 +5,6 @@
 "3d-view@^2.0.0":
   version "2.0.0"
   resolved "https://registry.yarnpkg.com/3d-view/-/3d-view-2.0.0.tgz#831ae942d7508c50801e3e06fafe1e8c574e17be"
-  integrity sha1-gxrpQtdQjFCAHj4G+v4ejFdOF74=
```

Comment:
please revert these changes, and we should have mysql and new sequelize updates in the yarn.lock

deleting integrity might mean your yarn is out of date?

Reply:
thx for that information ... and yes my yarn installation was super old.
I will update the yarn.lock file with the Integrity checksum.